01-filedir.md

```diff
@@ -342,13 +330,22 @@ Desktop/            north-pacific-gyre/ writing/
 
 `-a` stands for "show all";
 it forces `ls` to show us file and directory names that begin with `.`,
-such as `..` (which, if we're in `/Users/nelle`, refers to the `/Users` directory).
+such as `..` (which, if we're in `/Users/nelle`, refers to the `/Users` directory)
 As you can see,
 it also displays another special directory that's just called `.`,
 which means "the current working directory".
 It may seem redundant to have a name for it,
 but we'll see some uses for it soon.
-Finally, we also see a file called `.bash_profile`. This file usually contains settings to customize the shell (terminal). For this lesson material it does not contain any settings. There may also be similar files called `.bashrc` or `.bash_login`. The `.` prefix is used to prevent these configuration files from cluttering the terminal when a standard `ls` command is used.
+
+> ## Other Hidden Files {.callout}
+> 
+> In addition to the hidden directories `..` and `.`, you'll also see a file 
+> called `.bash_profile`. This file usually 
+> contains settings to customize the shell. For this lesson material 
```

Comment:
In the old data, there was a `.bash_profile` file in `/Users/nelle` that only contained comments.  Their `.bash_profile` might be different.  I think we could remove the "For this lesson material..." line.